pandas/io/json.py

```diff
@@ -123,32 +123,39 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
         file. For file URLs, a host is expected. For instance, a local file
         could be ``file://localhost/path/to/table.json``
 
-    orient
+    orient : string, indicating the expected format of the JSON input.
+        The set of allowed orients changes depending on the value
+        of the ``typ`` parameter.
```

Comment:
If we want to closely follow the numpy docstring standard, refering to other keywords would be with single backticks instead of double